iaso/tests/api/org_unit_change_requests/test_org_unit_change_requests.py

```diff
@@ -243,3 +248,68 @@ def test_delete_should_be_forbidden(self):
         change_request = m.OrgUnitChangeRequest.objects.create(org_unit=self.org_unit, new_name="Foo")
         response = self.client.delete(f"/api/orgunits/changes/{change_request.pk}/", format="json")
         self.assertEqual(response.status_code, 405)
+
+    def test_export_to_csv(self):
+        """
+        It tests the csv export for the org change requests list
+        """
+        group_1 = m.Group.objects.create(
+            name="Group 1", source_ref="qRsdUL2Oa4d", source_version=self.version, block_of_countries=False
+        )
+        group_2 = m.Group.objects.create(
+            name="Group 2", source_ref="KOSuvYwass8", source_version=self.version, block_of_countries=False
+        )
+        self.org_unit.groups.set([group_1, group_2])
+        org_unit_parent = m.OrgUnit.objects.create(name="parent")
+        self.org_unit.parent = org_unit_parent
+        self.org_unit.save()
+        self.org_unit.refresh_from_db()
+
+        m.OrgUnitChangeRequest.objects.create(org_unit=self.org_unit, new_name="Foo")
+        change_request = m.OrgUnitChangeRequest.objects.create(org_unit=self.org_unit, new_name="Bar")
+
+        change_request.created_by = self.user
+        change_request.updated_by = self.user
+        change_request.save()
+        change_request.refresh_from_db()
+
+        self.client.force_authenticate(self.user)
+
+        response = self.client.get("/api/orgunits/changes/export_to_csv/")
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(
+            response.get("Content-Disposition"),
+            "attachment; filename=review-change-proposals--" + datetime.datetime.now().strftime("%Y-%m-%d") + ".csv",
+        )
+
+        response_string = "\n".join(s.decode("U8") for s in response).replace("\r\n\n", "\r\n")
+        reader = csv.reader(io.StringIO(response_string), delimiter=",")
+        data = list(reader)
+        self.assertEqual(len(data), 4)
+
+        data_headers = data[1]
+        self.assertEqual(
+            data_headers,
+            self.org_unit_change_request_csv_columns,
+        )
+        first_data_row = data[2]
+        row_data = self.csv_row_data(change_request)
```

Comment:
```suggestion
        expected_row_data = self.csv_row_data(change_request)
```

`csv_row_data(change_request)` doesn't need to be a method since it's called only once.

You can hardode the expected data as a list, e.g.

```python
expected_row_data = [
    str(change_request.id),
    change_request.org_unit.name,
    …
]
```

It makes it crystal clear what we are expecting from the test.